### PR TITLE
Add CVE-2020-7354 and CVE-2020-7355 for Metasploit

### PR DESCRIPTION
**Pull request by todb-r7**
For the recently fixed Metasploit Pro bugs, discussed in issue 7104 in the security ticket queue.

@djordan-r7, can you take a look for spelling and grammar and sensibility? If you have any question, just hit me up on Slack.

### CODE REVIEW

**Comment by todb-r7** [comment]
btw I like how my signed commit totally failed here. whoops. I'll sign it when I push it up to MITRE.

File: 2020/7xxx/CVE-2020-7354.json [hunk 1]
```diff
@@ -1,18 +1,105 @@
...
+    ],
+    "data_format": "MITRE",
+    "data_type": "CVE",
+    "data_version": "4.0",
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "Cross-site Scripting (XSS) vulnerability in the 'host' field of a discovered scan asset in Rapid7 Metasploit Pro allows an attacker with a specially-crafted network service of a scan target store an XSS sequence in the Metasploit Pro console, which will trigger when the operator views the record of that scanned host in the Metasploit Pro interface. This issue affects Rapid7 Metasploit Pro version 4.17.1-20200427 and prior versions, and is fixed in Metasploit Pro version 4.17.1-20200514.\n\nSee also CVE-2020-7354, which describes a similar issue, but involving the generated 'notes' field of a discovered scan asset."
```
Comment (djordan-r7) [inline 2020/7xxx/CVE-2020-7354.json]: "...specially-crafted network service of a scan target to store an XSS sequence in the Metasploit Pro console..."

Comment (djordan-r7) [inline 2020/7xxx/CVE-2020-7354.json]: Shouldn't it be CVE-2020-7355 which affects the 'notes' field?

**Review by djordan-r7 — CHANGES_REQUESTED** [review]
@todb-r7 just a couple of minor changes :)

File: 2020/7xxx/CVE-2020-7354.json (hunk 1 above)
Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7354.json]: Derp, of course.

Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7354.json]: 7354 should be for 'host', 7355 should be 'notes' -- is this not the case? Copy pasting nearly the same thing always screws up something somewhere, but basically:

* CVE-2020-7354 should have the 'host' title and description, but also reference the 'notes' bug in 7355.
* CVE-2020-7355 should have the 'notes' title and description, but also reference the 'host' bug in 7354.

Is this not the case here? I've read it like 10 times and not seeing a mis-identification here (after my latest commit). You /should/ have the rights to edit the msp-xss branch if you can see something I'm not seeing.

Comment (todb-r7) [reply 2020/7xxx/CVE-2020-7354.json]: Re-reviewing, this looks good to me, now. Thanks for the review, @djordan-r7 !

**Review by todb-r7 — COMMENTED** [review]
Fixed and committed!

**Pull request merged**